test/Microsoft.AspNetCore.Razor.Language.Test/Legacy/CSharpDirectivesTest.cs

```diff
@@ -120,7 +120,7 @@ public void DirectiveDescriptor_UnderstandsStringTokens()
                     Factory.CodeTransition(),
                     Factory.MetaCode("custom").Accepts(AcceptedCharacters.None),
                     Factory.Span(SpanKind.Markup, " ", markup: false).Accepts(AcceptedCharacters.WhiteSpace),
```

Comment:
Left the string prefix as markup. This way when attempting to type a string token you don't get C# IntelliSense for things like `this` etc.